internal/tracker/tracker_test.go

```diff
@@ -481,224 +564,6 @@ func mustCreateFakeEmptyPipelineObject() runtime.Object {
 	return &pipeline
 }
 
-func TestFilterBundles(t *testing.T) {
```

Comment:
Removed this because I believe it is now covered by the newly added cases in TestTrack. I found this test a bit misleading because it didn't catch the issue where the `Repository` attribute is not set when loading data from YAML.